doc/examples/applications/plot_human_mitosis.py

```diff
@@ -186,15 +197,9 @@
 plt.show()
 
 #####################################################################
-# Additionally, we may use function `color.label2rgb` to overlay the original
-# image with the segmentation result, using transparency (alpha parameter).
-
-color_labels = color.label2rgb(segmented_cells, image, alpha=0.4, bg_label=0)
+# Make sure that the watershed algorithm has led to identifying more nuclei:
 
-fig, ax = plt.subplots(figsize=(5, 5))
-ax.imshow(color_labels)
-ax.set_title('Segmentation result over raw image')
-plt.show()
+assert segmented_cells.max() >= measure.label(cells).max()
```

Comment:
```suggestion
assert segmented_cells.max() > labeled_cells.max()
```
because of "more"...? And you can also just reuse `labeled_cells` here.